fpdf/table.py

```diff
@@ -382,7 +382,9 @@ def _render_table_cell(
         if not isinstance(text_align, (Align, str)):
             text_align = text_align[j]
         if i < self._num_heading_rows:
-            style = self._headings_style
+            # Always prioritize specified cell styles. If it's a heading row, then prioritize the
+            # heading style over default row style
+            style = cell.style or self._headings_style or row.style
```

Comment:
Would it maybe make sense to allow for partial overwriting of styles?
Let's say the heading style specifies bold emphasis and a blue background. Then the user gives the cell a style that only sets the text color to red. What a user might expect is to see a combination of the two.
You could do that by giving the `FontFace` class a method `this.combine(other)`, which creates a new instance using the values from "this" as the default, and only overwrites them with those values from "other" that aren't None.

Reply:
I think this is a good idea. Because it seemed there was the possibility that the default style could be None, I made `combine` a static function of `FontFace`.